doc/user/content/guides/reuse-topic-for-kafka-sink.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Topic reuse for Kafka sinks"
+description: "Enable topic reuse for Kafka sinks."
+weight: 50
+menu:
+  main:
+    parent: guides
+---
+
+{{< beta v0.9.0 />}}
+
+By default, Materialize creates new, distinct topics for sinks after each restart. To enable the reuse of the existing topic instead, Materialize must be able to reconstruct the prior history of the sinked object and all objects on which it is dependent--that is, events must have replayable timestamps--and must ensure that no other processes write to the output topic.
+
+This is currently available only for Kafka sources and the views based on them.
+
+When you create a sink, you must:
+
+- Enable the `reuse_topic` switch.
```

Comment:
Do we normally use `switch`?  Or should we call it a "`WITH` option"?

Reply:
Made it "option".